### PR TITLE
Appdata, desktop files and icon

### PR DESCRIPTION
**Pull request by nhlfr**

### CODE REVIEW

File: org.videolan.App.appdata.xml [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 <component type="desktop">
-  <id>com.CodeBlocks.App.desktop</id>
+  <id>org.videolan.App.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
-  <summary>IDE for C, C++ and Fortran</summary>
+  <summary>The cross-platform open-source multimedia framework, player and server</summary>
   <description>
     <p>
-     Code::Blocks is a highly extensible Integrated Development Environment (IDE) for the C, C++ and Fortran programming languages. It features support for a wide range of compilers, parallel builds, multi-target projects, and also provides an interface with GNU GDB.
+      VLC media player is a highly portable multimedia player and multimedia framework capable of reading most audio and video formats as well as DVDs, Audio CDs VCDs, and various streaming protocols.
```
Comment (chrskhl) [inline org.videolan.App.appdata.xml:8]: Need a comma after CDs

Comment (chrskhl) [reply org.videolan.App.appdata.xml:8]: Why not take the description as-is from the VLC site:

```
VLC is a free and open source cross-platform multimedia player and framework that plays most multimedia files as well as DVDs, Audio CDs, VCDs, and various streaming protocols.
```

**Review by chrskhl — CHANGES_REQUESTED** [review]
Just a couple small changes requested

**Review by lilic — COMMENTED** [review]
Not sure the indentation is correct here -> https://github.com/nhlfr/vlc-flatpak/blob/3dcee9f5df3a6d6d8559d81f1d54635c27d3c7f6/org.videolan.App.json#L1400-L1410

**Comment by chrskhl** [comment]
@LiliC right!

@nhlfr I was running it through `jq` to get proper/nice formatting.

**Comment by nhlfr** [comment]
@LiliC good point, I will fix it in another PR

**Comment by nhlfr** [comment]
@blixtra done

**Review by chrskhl — APPROVED** [review]
LGTM

**Pull request merged**